apps/scan/frontend/src/screens/internal_connection_problem_screen.tsx

```diff
@@ -25,20 +24,9 @@ function PrinterErrorMessage({
   if (printerStatus.state !== 'error') {
     return null;
   }
-  switch (printerStatus.type) {
-    case 'disconnected':
-      return appStrings.notePrinterDisconnected();
-    case 'hardware':
-      return appStrings.notePrinterHardwareError();
-    case 'supply-voltage':
-      return appStrings.notePrinterSupplyVoltageError();
-    case 'temperature':
-      return appStrings.notePrinterTemperatureError();
-    case 'receive-data':
-      return appStrings.notePrinterReceiveDataError();
-    default:
-      throwIllegalValue(printerStatus.type);
-  }
+  return printerStatus.type === 'disconnected'
+    ? appStrings.notePrinterDisconnected()
+    : appStrings.notePrinterHardwareError();
```

Comment:
Hmmm. I think that these messages are going to be so rare that I'd rather err on the side of specificity (giving the poll worker something to act on or tell their election manager from the precinct) than working to avoid scaring people. If the other decision makes sense to you and you want to stick to it, though, that's fine to me.

Reply:
@mattroe what do you think? I will leave this generic for now can always update in a future PR